src/builders/utils/uploader.ts

```diff
@@ -29,3 +37,37 @@ export async function uploadBuildToS3(ctx: IUploadCtx) {
     return fileLocation;
   }
 }
+
+export async function uploadBuildToTestFlight(ctx: IContext, options: IJobOptions) {
+  const fastlaneEnvVars = {
+    FASTLANE_SKIP_UPDATE_CHECK: 1,
+    FASTLANE_DISABLE_COLORS: 1,
+    CI: 1,
+    LC_ALL: 'en_US.UTF-8',
+    FASTLANE_PASSWORD: options.password,
+  };
+
+  const fastlaneArgs = [
+    'run upload_to_testflight',
+    'username:',
+    '"' + options.username + '"',
```

Comment:
```suggestion
    `"${options.username}"`,
```
Are double quotes necessary here?

Reply:
(fixed)  I was using the CLI documentation for running the command.